experimental/packages/otlp-transformer/README.md

```diff
@@ -15,8 +15,7 @@ To get started you will need to install a compatible OpenTelemetry API.
 
 ```sh
 npm install \
-    @opentelemetry/api \
-    @grpc/grpc-js # only required if you are using gRPC
+    @opentelemetry/api
```

Comment:
Nit: This could be inlined.

Reply:
Done in [aaac721](https://github.com/open-telemetry/opentelemetry-js/pull/2992/commits/aaac721a626b95acdac6b74f00035768ef0cac80). Thanks! :slightly_smiling_face: